lib/pf/Switch/Dell/N1500.pm

```diff
@@ -468,6 +483,64 @@ sub parseExternalPortalRequest {
     return \%params;
 }
 
+=head2
+
+Return Radius reply in the access-challenge
+
+=cut
+
+sub returnRadiusAdvanced {
```

Comment:
Same comment about reusing the code if possible

Reply:
It's not the same code as the ASA